server/build/babel/preset.js

```diff
@@ -7,10 +7,21 @@ const productionPlugins = isProduction ? [
   require.resolve('babel-plugin-transform-react-remove-prop-types')
 ] : []
 
+const es2015Config = {}
+
+// Do not transform ES2015 modules if we are inside Next.js
+// That's because webpack2 knows how to handle it.
+// (And that's how it can do code splitting)
+//
+// But in other environements like Jest, we should transform it.
+if (process.env.INSIDE_NEXT) {
```

Comment:
@nkzawa I mean this: https://github.com/zeit/next.js/issues/1042
See: https://github.com/angel200sdnot/sample_repo/blob/master/package.json#L17

With that, he/she is transpiling server.js using `next/babel`. It works before not now.